src/pages/Home/components/Portfolio/Prompt/Prompt.tsx

```diff
@@ -142,6 +142,21 @@ export function Prompt() {
           content: `Success! The new active account is ${accountId}`,
         };
       },
+      switchNetwork: async ({ networkId }: { networkId: string }) => {
+        const newActiveNetwork = networks.find((networkItem) => {
+          return (
+            networkItem.caipId === networkId ||
+            networkItem.chainId === parseInt(networkId)
+          );
+        });
```

Comment:
You can use `getNetwork()` from `useNetworkContext`, it will take care of the details like using `caipId` or `chainId`, etc.